tests/protractor/utils.js

```diff
@@ -178,5 +184,10 @@ module.exports = {
     `http://${constants.API_HOST}:${constants.API_PORT}/${constants.DB_NAME}`,
 
   getLoginUrl: () =>
-    `http://${constants.API_HOST}:${constants.API_PORT}/${constants.DB_NAME}/login`
+    `http://${constants.API_HOST}:${constants.API_PORT}/${constants.DB_NAME}/login`,
+    takeScreenshot: function (filename) {
+    browser.takeScreenshot().then(function (png) {
```

Comment:
indentation; spacing around `function`; use of `function` instead of ES6 notation